backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -63,7 +63,7 @@
           end
         end
 
-        # Click "ok" on confirmation dialog
+        expect(page).to have_content("YOUR ORDER IS EMPTY") # wait for page refresh
```

Comment:
The capybara matchers both wait (up to 2 seconds by default) until the condition they are looking for is true. This not_to have_content returns almost immediately, since the row is hidden in the click event handler. Meanwhile, the phantomjs browser continues running the ajax request to delete the item and then refreshes the page.